test/ttmlir/Dialect/TTIR/test_grid_set.mlir

```diff
@@ -3,7 +3,7 @@
 module attributes {} {
   func.func @forward(%arg0: tensor<64x128xf32>, %arg1: tensor<64x128xf32>) -> tensor<64x128xf32> {
     %0 = tensor.empty() : tensor<64x128xf32>
-    // CHECK: #[[LAYOUT_1:.*]] = #tt.layout<(d0, d1) -> (d0, d1), undef, <8x8>, memref<8x16xf32, #dram>, interleaved>
+    // CHECK: #[[LAYOUT_1:.*]] = #tt.layout<(d0, d1) -> (d0, d1), undef, <1x1>, memref<64x128xf32, #dram>, interleaved>
```

Comment:
Why is this changed to 1x1?

Reply:
@nsmithtt @jnie-TT Do we have any plans on how to infer computation grid for OPs which operate with DRAM data?

Reply:
This seems to be different on an op per op basis and is discussed in #450, #559 and is being addressed in #605 .